bundles/org.openhab.binding.shelly/README.md

```diff
@@ -384,10 +506,41 @@ The Shelly 4Pro provides 4 relays and 4 power meters.
 |          |lastPower2   |Number   |yes      |Energy consumption in Watts for a round minute, 2 minutes ago                    |
 |          |lastPower3   |Number   |yes      |Energy consumption in Watts for a round minute, 3 minutes ago                    |
 |          |totalKWH     |Number   |yes      |Total energy consumption in Watts since the device powered up (resets on restart)|
-|          |timestamp    |String   |yes      |Timestamp of the last measurement                                                |
+|          |lastUpdate   |String   |yes      |Timestamp of the last measurement                                                |
+
+### Shelly Dimmer2 (thing-type: shellydimmer2)
+
+|Group     |Channel      |Type     |read-only|Description                                                                      |
+|----------|-------------|---------|---------|---------------------------------------------------------------------------------|
+|relay     |brightness   |Dimmer   |r/w      |Currently selected brightness.                                                   |
+|          |input1       |Switch   |yes      |State of Input 1 (S1)                                                            |
+|          |input2       |Switch   |yes      |State of Input 2 (S2)                                                            |
+|          |autoOn       |Number   |r/w      |Sets a  timer to turn the device ON after every OFF command; in seconds          |
+|          |autoOff      |Number   |r/w      |Sets a  timer to turn the device OFF after every ON command; in seconds          |
+|          |button       |Trigger  |yes      |Event trigger with payload SHORT_PRESSED or LONG_PRESSED (FW 1.5.6+)             |
+|status    |loaderror    |Switch   |yes      |Last error, "no" if none                                                         |
+|          |overload     |Switch   |yes      |Overload condition detected, switch dimmer off or reduce load!                   |
+|          |overtemperature |Switch|yes      |Internal device temperature over maximum. Switch off, check physical installation|
+|meter     |currentWatts |Number   |yes      |Current power consumption in Watts                                               |
+|          |lastPower1   |Number   |yes      |Energy consumption in Watts for a round minute, 1 minute  ago                    |
```

Comment:
changed to "Accumulated energy consumption in Watts for the full last minute "
lastPower2 and lastPower3 have been removed, because nobody used it (needs to be documented)

Reply:
Energy is Wh. You might mean "Average power consumption".